lib/course.rb

```diff
@@ -1 +1,27 @@
+require './lib/student'
 
+class Course
+  attr_reader :name, :capacity, :students
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+    @full = false
+  end
+
+  def full?
+    if @students.size < @capacity
+      @full = false
+    else
+      @full = true
+    end
```

Comment:
```suggestion
@students.size >= @capacity
```
This is a little verbose. We don't really need the instance var on line 9, nor do we need to set it here. Comparing the number of students to capacity is sufficient to return the result we need here.